uiowa/uiowa
# Update hook to fix `table` classes

Resolves #3447 

# How to test

* `blt sync --site transportation.uiowa.edu`
* `drush @transportation.local uli admin/content`
* `yarn workspace uids_base gulp --development`
* https://transportation.local.drupal.uiowa.edu/cambus/routes-schedules-maps leads to a lot of pages with tables.
* Check this page to make sure that the corrected table class `table--hover-highlight` is still there.
* Add `table--width-default` to a smaller sized table like on the bottom of https://transportation.local.drupal.uiowa.edu/cambus/routes-schedules-maps/va-loop-shuttle-route and make sure the table does not stretch full-width across the page.
